srcpkgs/webkit2gtk/template

```diff
@@ -142,11 +165,75 @@ webkit2gtk-devel_package() {
 	depends="gtk+3-devel libsoup-devel ${sourcepkg}>=${version}_${revision}"
 	short_desc+=" - development files"
 	pkg_install() {
-		vmove usr/include
-		vmove usr/lib/pkgconfig
+		vmove usr/include/webkitgtk-4.0
+		vmove "usr/lib/pkgconfig/*-4.0.pc"
+		if [ "$build_option_gir" ]; then
+			vmove "usr/share/gir-1.0/*-4.0.gir"
+			vmove "usr/share/gtk-doc/html/*-4.0"
+		fi
+		vmove "usr/lib/*-4.0.so"
+	}
+}
+
+webkit2gtk-common_package() {
+	short_desc+="GTK port of the WebKit2 browser engine - common files"
+	pkg_install() {
+		vmove usr/share/locale/
+		vmove usr/share/licenses/
+	}
+}
+
+libwebkit2gtk41_package() {
+	depends="${depends}"
```

Comment:
I am not sure if this is safe, are you sure this will always use $depends from the main package? Either way, it's only webkit2gtk-common, so I think it might make sense to just write that.

Reply:
It also needs bubblewrap and xdg-dbus-proxy